clients/githubrepo/branches.go

```diff
@@ -190,7 +190,13 @@ func copyAdminSettings(src *branchProtectionRule, dst *clients.BranchProtectionR
 	copyBoolPtr(src.DismissesStaleReviews, &dst.RequiredPullRequestReviews.DismissStaleReviews)
 	if src.RequiresStatusChecks != nil {
 		copyBoolPtr(src.RequiresStatusChecks, &dst.CheckRules.RequiresStatusChecks)
-		copyBoolPtr(src.RequiresStrictStatusChecks, &dst.CheckRules.UpToDateBeforeMerge)
+		// Workaround for GitHub GraphQL bug https://github.com/orgs/community/discussions/59471
```

Comment:
Can you create a tracking bug on scorecard repo and update the comment `TODO(number): Update when grahpQL big is fixed`

Reply:
Yes!